demo/get_erc721_transfer_history_blocking.cc

```diff
@@ -0,0 +1,72 @@
+#include "include/extra-cpp-bindings/src/lib.rs.h"
+#include "include/rust/cxx.h"
+#include <iostream>
+using namespace com::crypto::game_sdk;
+
+// Read CronoScan api key in env
+const rust::String CRONOSCAN_API_KEY = getenv("CRONOSCAN_API_KEY");
+
+int main(int argc, char *argv[]) {
+  // Get a list of "ERC721 - Token Transfer Events" by Address
+  // Returns up to a maximum of the last 10000 transactions only
+  // https://cronoscan.com/tokentxns-nft?a=0x668f126b87936df4f9a98f18c44eb73868fffea0
+  rust::Vec<RawTxDetail> erc721_txs = get_erc721_transfer_history_blocking(
+      "0x668f126b87936df4f9a98f18c44eb73868fffea0", "", QueryOption::ByAddress,
+      CRONOSCAN_API_KEY);
+
+  for (rust::Vec<RawTxDetail>::iterator ptr = erc721_txs.begin();
```

Comment:
Vec<RawTxDetail>::iterator looks simpler

Reply:
also consider for ..in..

Reply:
ok

Reply:
done